saleor/graphql/webhook/resolvers.py

```diff
@@ -14,13 +17,16 @@
 from .types import Webhook, WebhookEvent
 
 
-def resolve_webhook(info: ResolveInfo, id, app):
+def resolve_webhook(info, id, app):
```

Comment:
Minor thing but here the port removes typing for the `info` field, but it should stay (it was added in 3.15).